images/prow-tests/cloudbuild.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+- name: gcr.io/cloud-builders/docker
+  args: ['build', '--no-cache', '--pull', '-t',
+         '${_IMG}:${_TAG}',
+         '-f', 'images/prow-tests/Dockerfile', '.']
+- name: gcr.io/cloud-builders/docker
+  args: [ 'tag',
+          '${_IMG}:${_TAG}',
+          '${_IMG}:beta']
+- name: gcr.io/cloud-builders/docker
```

Comment:
This step depends on previous step, should you use `waitFor`?

Reply:
waitFor is only needed if you are doing local parallelism.  each step implicitly waits for the previous step in a vanilla sequence

Reply:
that's right, nvm then